Makefile

```diff
@@ -317,10 +317,11 @@ validate: lint .gitvalidation validate.completions man-page-check swagger-check
 
 .PHONY: validatepr
 validatepr:
-	$(PODMANCMD) run --rm --env HOME=/root \
-		-v $(CURDIR):/var/tmp/go/src/github.com/containers/podman \
+	$(PODMANCMD) run --rm \
+		-v $(CURDIR):/go/src/github.com/containers/podman \
 		--security-opt label=disable \
-		quay.io/libpod/fedora_podman:latest  \
+		-w /go/src/github.com/containers/podman \
+		quay.io/libpod/validatepr:latest  \
```

Comment:
This image is private.
Second, how/where is the is image build and updated regularly?

Reply:
we will add a gh hook once we have all the various incantations worked out.  by then, it might be that the official image be multiarch?  And thanks for the reminder about the private vs public.  Fixed.

Reply:
I am fine to merge without automation, however I am not fine merging without docs on how the image is created. I assume contrib/validatepr/Containerfile is the source? Can you add a comment in the Makefile to link this and add this to the description on quay.io as well. In this case anyone can figure out where this came from in case you are not around.